src/components/bs5/header/header.hbs

```diff
@@ -174,45 +174,109 @@
+                                        <span class="qld__header__cta-link-icon">
+                                            <i class="{{icon-root}}#{{icon.value}}"></i>
+                                        </span>
+                                    {{/if}}
+                                    <span class="qld__header__cta-link-text">{{text.value}}</span>
+                                </a>
+
+                                {{#if dropdown_enabled}}
+                                    <ul class="dropdown-menu dark" aria-labelledby="dropdown{{id}}">
+
+                                        {{#isType dropdown_options.dropdown_type "list"}}
+                                            {{#if dropdown_options.dropdown_config}}
+                                                {{#each dropdown_options.dropdown_config.groups}}
+                                                    <li class="qld__header__cta-list">
+                                                        <a class="" href="{{href}}">
+                                                            {{label}}
+                                                            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                                                <path d="M18.8592 12.6135L13.6659 17.6633C13.5721 17.7781 13.4156 17.8068 13.2905 17.8068C13.1654 17.8068 13.0402 17.7781 12.9464 17.692C12.7274 17.5199 12.7274 17.233 12.9151 17.0608L17.295 12.757H5.50056C5.21899 12.757 5 12.5562 5 12.3266C5 12.0971 5.21899 11.8389 5.50056 11.8389H17.295L12.9151 7.56375C12.7274 7.3916 12.7274 7.10468 12.9464 6.93253C13.1654 6.76038 13.4782 6.76038 13.6659 6.96122L18.8592 12.011C19.0469 12.1832 19.0469 12.4414 18.8592 12.6135Z" fill="#FFE500"/>
+                                                            </svg>
+                                                        </a>
+                                                    </li>
+                                                {{/each}}
+                                            {{/if}}
+                                        {{/isType}}
+
+                                        {{#isType dropdown_options.dropdown_type "form"}}
+                                            {{#if dropdown_options.dropdown_config.groups}}
+                                                {{#each dropdown_options.dropdown_config.groups}}
+
+                                                    {{#if heading}}<h4 class="mt-2">{{heading}}</h4>{{/if}}
+                                                    {{#if content}}<p>{{content}}</p>{{/if}}
+
+                                                    {{#isType actionType "button"}}
+                                                        <a class="btn btn-primary" href="{{action.button_action}}" {{#isType
+                                                            action.button_targetAttr "_blank" }} target="_blank" {{/isType}}>
```

Comment:
Can we use  _target="{{action.button_targetAttr}}"_ here.

Thinking it would allow other values to be used also, e.g. _self, _parent. If action.button_targetAtr is "", browser will just treat link normally anyway.